src/pineko/evolve.py

```diff
@@ -71,11 +71,25 @@ def write_operator_card(pineappl_grid, default_card, card_path, xif):
     q2_grid = (xif * xif * muf2_grid).tolist()
     operators_card["targetgrid"] = x_grid.tolist()
     operators_card["Q2grid"] = q2_grid
+    if is_integrability(pineappl_grid.key_values()):
+        operators_card["interpolation_polynomial_degree"] = 1
+        x_grid_int = copy.deepcopy(x_grid.tolist())
+        x_grid_int.append(1.0)
+        operators_card["interpolation_xgrid"] = list(x_grid_int)
     with open(card_path, "w", encoding="UTF-8") as f:
         yaml.safe_dump(operators_card, f)
     return x_grid, q2_grid
 
 
+def is_integrability(grid_key_values):
+    """Check if the grid is an integrability grid."""
+    try:
```

Comment:
why not just `return "integrability_version" in grid_key_values`? and since it then becomes a one-line function used a single time, I'd suggest to inline ..

Reply:
Because if the grid is not integrability, it won't have the key `integrability_version` at all and so that would result in an error.

Reply:
That is the reason for the `try/except`

Reply:
hum? that's exactly what `in` does, isn't it?
```
In [1]: d = {"a": 1}

In [2]: "a" in d
Out[2]: True

In [3]: "b" in d
Out[3]: False
```

Reply:
yes right :)